content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -201,223 +201,29 @@ If the majority of etcd members have permanently failed, the etcd cluster is con
-script works as follows:
-
-1. Verify that the detected version is 3.0.x with the v3 API, and the
-   desired version is 2.2.1 with the v2 API. We don’t support any other rollback.
-1. If so, we run the custom tool provided by etcd team to do the offline
-   rollback. This tool reads the v3 formatted data and writes it back to disk
-   in v2 format.
-1. Finally update the contents of the version file.
-
-### Upgrade procedure
-Simply modify the command line in the etcd manifest to:
-
-1. Run the migration script. If the previously run version is already in the
-   desired version, this will be no-op.
-1. Start etcd in the desired version.
-
-Starting in Kubernetes version 1.6, this has been done in the manifests for new
-Google Compute Engine clusters. You should also specify these environment
-variables. In particular, you must keep `STORAGE_MEDIA_TYPE` set to
-`application/json` if you wish to preserve the option to roll back.
-
-```
-TARGET_STORAGE=etcd3
-ETCD_IMAGE=3.0.17
-TARGET_VERSION=3.0.17
-STORAGE_MEDIA_TYPE=application/json
-```
-
-To roll back, use these:
-
-```
-TARGET_STORAGE=etcd2
-ETCD_IMAGE=3.0.17
-TARGET_VERSION=2.2.1
-STORAGE_MEDIA_TYPE=application/json
-```
-
-{{< note >}}
-**Note:** This procedure upgrades from 2.x to 3.x.  Version `3.0.17` is not recommended for running in production (see [prerequisites](#prerequisites) for minimum recommended etcd versions).
-{{< /note >}}
-
-## Notes for etcd Version 2.2.1
-
-### Default configuration
-
-The default setup scripts use kubelet's file-based static pods feature to run etcd in a
-[pod](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/gce/manifests/etcd.manifest). This manifest should only
-be run on master VMs. The default location that kubelet scans for manifests is
-`/etc/kubernetes/manifests/`.
-
-### Kubernetes's usage of etcd
-
-By default, Kubernetes objects are stored under the `/registry` key in etcd.
-This path can be prefixed by using the [kube-apiserver](/docs/admin/kube-apiserver) flag
-`--etcd-prefix="/foo"`.
-
-`etcd` is the only place that Kubernetes keeps state.
-
-### Troubleshooting
-
-To test whether `etcd` is running correctly, you can try writing a value to a
-test key. On your master VM (or somewhere with firewalls configured such that
-you can talk to your cluster's etcd), try:
-
-```shell
-curl -X PUT "http://${host}:${port}/v2/keys/_test"
-```
+As of Kubernetes v1.13.0, etcd2 is no longer supported as a storage backend for
+new or existing Kubernetes clusters. The timeline for Kubernetes support for
+etcd2 and etcd3 is as follows: 
+
+- Kubernetes v1.0: etcd2 only
+- Kubernetes v1.5.1: etcd3 support added, new clusters still default to etcd2
+- Kubernetes v1.6.0: new clusters created with `kube-up.sh` default to etcd3
+- Kubernetes v1.9.0: deprecation of etcd2 storage backend announced
+- Kubernetes v1.13.0: etcd2 storage backend removed
+
+In the unlikely scenario that your Kubernetes cluster is older than v1.13.0 and
```

Comment:
is the thought that the passage of time will make this statement more and more true? The text from the 1.13 release notes might be clearer (you know you are in this situation from the `--storage-backend` flag):

> Before upgrading a v1.12.x kube-apiserver using `--storage-backend=etcd2` to v1.13.x, etcd v2 data MUST be migrated to the v3 storage backend, and kube-apiserver invocations changed to use --storage-backend=etcd3.

Reply:
the fact that the v1.13.x server will refuse to start with `--storage-backend=etcd2` with a `etcd2 is no longer a supported storage backend` error is useful for pushing upgraders in that situation to look for docs. Maybe include that error message somewhere in this doc to drive searchers here.

Reply:
Agree this was the wrong tone to take, thanks for confirming, and your wording is much better